test/unit/model_runners/extractors/test_create_extractor.py

```diff
@@ -9,9 +9,16 @@ def test_create_extractor():
     assert isinstance(create_extractor(model_accept_type=MIME_TYPE_JSON, output_location="output"), JsonExtractor)
 
 
-def test_create_extractor_jumpstart():
+@pytest.mark.parametrize(
```

Comment:
can you also choose a model that doesn't have default payloads to test (e.g. `xgboost-regression-snowflake`)